fuse_optimizers/src/fixed_lag_smoother.cpp

```diff
@@ -425,6 +435,34 @@ void FixedLagSmoother::processQueue(fuse_core::Transaction& transaction, const r
 }
 
 bool FixedLagSmoother::resetServiceCallback(std_srvs::Empty::Request&, std_srvs::Empty::Response&)
+{
+  stop();
+  start();
+
+  return true;
+}
+
+bool FixedLagSmoother::stopServiceCallback(std_srvs::Empty::Request&, std_srvs::Empty::Response&)
+{
+  stop();
+  return true;
+}
+
+bool FixedLagSmoother::startServiceCallback(std_srvs::Empty::Request&, std_srvs::Empty::Response&)
+{
+  start();
+  return true;
+}
```

Comment:
The request and response are not used. I assume this is only a stub for future filling in.

Reply:
Not a stub, there's just no information currently needed for a start or stop, the same as how the reset callback also doesn't use the request or response fields.